packages/components/src/autocomplete/get-default-use-items.tsx

```diff
@@ -61,7 +68,7 @@ export default function getDefaultUseItems( autocompleter ) {
 			const { options, isDebounced } = autocompleter;
 			const loadOptions = debounce(
 				() => {
-					const promise = Promise.resolve(
+					const promise: DebouncedPromise = Promise.resolve(
```

Comment:
Ha, great minds think alike. I'd intended to comment on the declaration for this in `types.ts` because I was struggling with the naming. `Debounced` didn't feel totally accurate, but `Cancelable` also felt like it lacked context, so I kept going back and forth.

I'm happy with `CancelablePromise`, thanks for spotting this :) 86a102f713